src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -899,32 +899,16 @@ Here's a step-by-step explanation of how topology correlation works:
 
 4. The incidents are then correlated and the issues are merged together.
 
-### Create your topology graph [#create-topology-graph]
-
-To create and maintain your topology graph use the aiTopologyCollector APIs in [Nerdgraph](https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to create vertices and connect them together with edges. Note that deleting a vertex will remove all edges connected with it.
-
-![New Relic topology explained 4](./images/topology-4.png "topology-4.png")
-
-Here are definitions of key topology concepts:
-
-* **Vertices:** You can think of a vertex as an entity and the source from which your incident events are coming from or describing the problematic symptom about. To create a vertex, you need to define these attributes:
-  * Name: The unique name of the vertex such as an application, service, or host name. This is value must be unique within the graph.
-  * Defining attributes: A vertex requires a set of special attributes (key/value pairs) to allow it to be associated with incoming incident events. Unique identifiers that appear on all incidents are recommended, such as entity GUIDs or other IDs.
-  * Vertex class (optional): An optional class can be set such as `application`, `host`, or `datastore`. This allows your decision logic to restrict the vertices the "topologically dependent" operator traverses to only those you define. For example, "test if vertex A is connected to vertex B traversing only application type vertices".
-* **Edges:** An edge is simply a connection between two vertices. Use edges to describe how everything is connected together. To create an edge, you need to define these attributes:
-  * `from vertex name`: The name of the vertex you're starting to define an edge for.
-  * `to vertex name`: The name of the vertex at which you're ending the edge.
-  * `directed`: This is a boolean indicating that the edge connection goes only in one direction. The direction, if thought of as an arrow, starts at the `from vertex name` and points at the `to vertex name`. This is currently used in the dependent operator. 
-
-![New Relic topology explained 5](./images/topology-5.png "topology-5.png")
-
 ### Add attributes to incident events [#add-attributes]
 
-Incidents are connected to vertices using a vertex's defining attributes. In the example topology graph under [Topology explained](#topology-explained), each vertex has a defining attribute "CID" with a unique value. Next, we find a vertex that matches the attribute of an incident.
+Incidents are connected to vertices using a vertex's defining attributes (in the example topology under [Topology explained](#topology-explained), each vertex has a defining attribute "CID" with a unique value). Next, a vertex is found that matches the attribute of an incident.
```

Comment:
Trying to steer away from passive voice... How about "Next, Applied Intelligence finds a vertex that matches the attribute..."